redfish-core/lib/log_services.hpp

```diff
@@ -1593,6 +1593,20 @@ inline void requestRoutesSystemLogServiceCollection(App& app)
             "/redfish/v1/Systems/system/LogServices/HostLogger";
         logServiceArray.emplace_back(std::move(hostlogger));
 #endif
+
+#ifdef BMCWEB_ENABLE_REDFISH_DBUS_LOG_ENTRIES
+        Privileges effectiveUserPrivileges =
+            redfish::getUserPrivileges(*req.session);
+
+        if (isOperationAllowedWithPrivileges({{"ConfigureManager"}},
+                                             effectiveUserPrivileges))
+        {
+            logServiceArray.push_back(
+                {{"@odata.id", "/redfish/v1/Systems/system/LogServices/"
+                               "CELog"}});
```

Comment:
1. New clang-format recommends not to break a string
`"/redfish/v1/Systems/system/LogServices/CELog"`

2. https://github.com/ibm-openbmc/bmcweb/blob/1060/COMMON_ERRORS.md#13-complete-replacement-of-the-response-object suggests

```
           nlohmann::json::object_t item;
          item["@odata.id"] = "/redfish/v1/Systems/system/LogServices/CELog";
            logServiceArray.emplace_back( std::move(item) );
```

Reply:
will fix.

Reply:
done